var/spack/repos/builtin/packages/openkim-models/package.py

```diff
@@ -19,19 +19,23 @@ class OpenkimModels(CMakePackage):
        compatible with the kim-api package.
     """
     homepage = "https://openkim.org/"
-    url      = "https://s3.openkim.org/archives/collection/openkim-models-2019-03-29.txz"
+    url      = "https://s3.openkim.org/archives/collection/openkim-models-2019-07-25.txz"
 
     extends('kim-api')
 
-    version('2019-03-29', sha256='053dda2023fe4bb6d7c1d66530c758c4e633bbf1f1be17b6b075b276fe8874f6')
+    version(
+       '2019-07-25',
+       sha256='50338084ece92ec0fb13b0bbdf357b5d7450e26068ba501f23c315f814befc26')
+    version(
+       '2019-03-29',
+       sha256='053dda2023fe4bb6d7c1d66530c758c4e633bbf1f1be17b6b075b276fe8874f6')
 
     def cmake_args(self):
         args = []
         args.append(('-DKIM_API_MODEL_DRIVER_INSTALL_PREFIX={0}'
                      + '/lib/kim-api/model-drivers').format(prefix))
-        args.append('-DKIM_API_MODEL_INSTALL_PREFIX={0}/lib/kim-api/models'
-                    .format(prefix))
-        args.append('-DKIM_API_SIMULATOR_MODEL_INSTALL_PREFIX={0}/lib/'
-                    'kim-api/simulator-models'
-                    .format(prefix))
+        args.append(('-DKIM_API_PORTABLE_MODEL_INSTALL_PREFIX={0}'
+                     + '/lib/kim-api/portable-models').format(prefix))
+        args.append(('-DKIM_API_SIMULATOR_MODEL_INSTALL_PREFIX={0}'
+                     + '/lib/kim-api/simulator-models').format(prefix))
```

Comment:
You changed the arg names and directories. Does this work for both `2019-03-29` _and_ `2019-07-25`? If not, you'll have to use different args for different versions.

Reply:
Ah Ha!  Actually this depends on the kim-api package version.  How do I go about setting these args depending on the version of its dependency?  (Thanks for your help!)

Reply:
You can do something like:
```python
if self.spec.satisfies('@1.2.3:'):
    ...
else:
    ...
```

Reply:
Well....  I've made it hard for myself...
How can I limit the versions of a dependency that are compatible with a specific version of the package?

In words, I want to specify: "versions `openkim-models@2019-07-25:` must be used with version `kim-api@2.1.0:` " and "version `openkim-models@2019-03-29` must be used with version `kim-api@:2.0.2` "

Is that possible?

Reply:
```python
depends_on('kim-api@2.1.0:', when='@2019-07-25:')
depends_on('kim-api@:2.0.2', when='@2019-03-29')
```